Sources/LiveViewNative/Views/Images/Image.swift

```diff
@@ -28,6 +28,7 @@ import SwiftUI
 @_documentation(visibility: public)
 #endif
 struct Image: View {
+    @Attribute("resizable") private var resizable: Bool
```

Comment:
Can you add a doc comment (and the related dance for `@_documentation`) for this?

Reply:
Done! I noticed the compiler directive from other documentation so I went ahead and copied that, but not sure if it's needed. Also feel free to suggest changes to the documentation text I wrote, not sure if there are any rules / conventions around that.